src/main/java/com/conveyal/analysis/util/ProgressInputStream.java

```diff
@@ -0,0 +1,57 @@
+package com.conveyal.analysis.util;
```

Comment:
Should we create a `progress` package with the corresponding files created specifically for task management and progress reporting? Having them circularly referenced across packages like this is not ideal to me.

Reply:
Discussed in last call - I agree but we'll do this as part of a bigger reshuffle pulling analysis backend out above the r5 package tree.